web/crux/src/app/health/health.controller.ts

```diff
@@ -4,16 +4,65 @@ import { CruxHealthController, CruxHealthControllerMethods, HealthResponse } fro
 import CommonGrpcErrorInterceptor from 'src/interceptors/grpc.error.interceptor'
 import GrpcLoggerInterceptor from 'src/interceptors/grpc.logger.interceptor'
 import PrismaErrorInterceptor from 'src/interceptors/prisma-error-interceptor'
+import {
+  HealthCheckRequest,
+  HealthCheckResponse,
+  HealthCheckResponse_ServingStatus,
+  HealthController as ProtoHealthController,
+  HealthControllerMethods,
+} from 'src/grpc/protobuf/proto/health'
+import { Observable } from 'rxjs'
+import { NotFoundException, NotImplementedException } from 'src/exception/errors'
 import HealthService from './health.service'
+import HealthMapper from './health.mapper'
 
 @Controller()
 @CruxHealthControllerMethods()
+@HealthControllerMethods()
 @UseInterceptors(GrpcLoggerInterceptor, CommonGrpcErrorInterceptor, PrismaErrorInterceptor)
-export default class HealthController implements CruxHealthController {
-  constructor(private readonly service: HealthService) {}
+export default class HealthController implements CruxHealthController, ProtoHealthController {
+  constructor(private readonly service: HealthService, private mapper: HealthMapper) {}
 
   @AuditLogLevel('disabled')
   async getHealth(): Promise<HealthResponse> {
     return this.service.getHealth()
   }
+
+  @AuditLogLevel('disabled')
+  async check(request: HealthCheckRequest): Promise<HealthCheckResponse> {
+    if (!request.service || request.service === '') {
```

Comment:
Why do you need to check the `request.service` property falseness?

Reply:
I used a standard tool to check the implementation and it sends nothing to check the service health (the standard says empty string so I implemented both).